packages/web-client/src/graph.ts

```diff
@@ -133,17 +138,16 @@ export const graph = (baseURI: string, axiosClient: AxiosInstance): Graph => {
       deleteUser: (userId: string) => userApiFactory.deleteUser(userId),
       listUsers: (orderBy?: any, filter?: string) =>
         usersApiFactory.listUsers(
-          0,
-          0,
```

Comment:
Unrelated, but some parameters seems to have gone after updating the graph client.